src/structures/Player.ts

```diff
@@ -119,7 +124,9 @@ export class Player {
     if (options.textChannel) this.textChannel = options.textChannel;
 
     const node = this.manager.nodes.get(options.node);
-    this.node = node || this.manager.leastLoadNodes.first();
+    if (node) this.node = node
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585535)
```suggestion
    if (node) {this.node = node}
```